### ISSUE TITLE
Feature Request: File operation continuity across device group shares and remote client file manager.

### ISSUE THREAD

**Issue by MatinatorX** [Ylianst/MeshCentral#214]
MeshCentral device group shares are a great way to store backups and other rfiles specific to that remote client. However, from what I can tell we currently have to login to the server's web portal on the remote client to be able to upload or download files to and from the device group shares.

It would be great if the file operation commands worked across the device group shares and the remote client file manager, so we could more easily transfer files between the two. Even better might be a "download to remote client" and a "upload from remote client" button/dialogue within the file manager.

**Comment by Ylianst** [comment]
Agreed, that would be nice. Lots going on, but certainly something that needs to be done at some point.
